docs/metrics-howto.md

```diff
@@ -60,6 +60,31 @@
 
 1.  Restart Prometheus.
 
+## Monitoring workers
+
+To monitor a Synapse installation using
+[workers](https://github.com/matrix-org/synapse/blob/master/docs/workers.md),
+every worker needs to be monitored independently, in addition to
+the main homeserver process. This is because workers don't send
+their metrics to the main homeserver process, and therefore
+expose their own metrics themselves (if they are configured to
+do so).
+
+To allow collecting metrics from a worker, you need to add a
+`metrics` listener to its configuration, by adding the following
+under `worker_listeners`:
+
+```yaml
+ - type: metrics
+   bind_address: ''
+   port: 9101
+```
+
+Feel free to change the `bind_address` and `port` parameters
+as long as the resulting listener can be reached by prometheus.
```

Comment:
This sentence is a bit odd for documentation, in my opinion. Also should we mention to avoid port conflicts between multiple workers?

Reply:
Ah, right, I copied it in a brainless way from an issue comment haha. And yes. Fixed in https://github.com/matrix-org/synapse/pull/7357/commits/236dbba7e323e38f6bb4264e75970823cc8d7bec